### DISCUSSION TITLE
Doc/urql in nodejs

### DISCUSSION THREAD

**Discussion by hanakannzashi** [urql-graphql/urql#2724] [Q&A]
[Doc](https://formidable.com/open-source/urql/docs/basics/core/) seems has something wrong since this page is for nodejs
<img width="885" alt="截屏2022-10-09 下午12 17 01" src="https://user-images.githubusercontent.com/95295094/194737825-73237f27-b42e-4e56-9775-cc2da9bfb827.png">
Should we add option `fetch` here, using `cross-fetch` ?

**Comment by JoviDeCroock** [comment]
Yes we might need to add an extract that Node > 17.5 will just work but everything before needs a package like `node-fetch`
